python/aitubertalk/rooms/types.py

```
@@ -0,0 +1,74 @@
+"""
+Type definitions for room management module
+"""
+
+from dataclasses import dataclass, field
+from datetime import datetime
+from typing import Dict, List, Optional
```

Ruff (0.11.9)

`typing.Dict` imported but unused

Remove unused import: `typing.Dict`

(F401)